unic/char/range/src/step.rs

```diff
@@ -0,0 +1,40 @@
+use std::char;
```

Comment:
Oh, I thought we have a `char.rs` here next to `step.rs`. You're right, there's no conflict and it's fine. #aftermidnightreview